modules/core/02-client/keeper/grpc_query.go

```diff
@@ -204,11 +204,13 @@ func (q Keeper) UpgradedClientState(c context.Context, req *types.QueryUpgradedC
 		return nil, status.Error(codes.InvalidArgument, "empty request")
 	}
 
-	if err := host.ClientIdentifierValidator(req.ClientId); err != nil {
-		return nil, status.Error(codes.InvalidArgument, err.Error())
+	if req.PlanHeight < 1 {
+		return nil, status.Error(codes.InvalidArgument, "non positive plan height")
 	}
 
 	ctx := sdk.UnwrapSDKContext(c)
+	ctx = ctx.WithBlockHeight(req.PlanHeight)
```

Comment:
Hmm, we might want to be able to query if the client state is set before the plan height